schematic/models/metadata.py

```diff
@@ -126,12 +127,20 @@ def getModelManifest(self, title: str, rootNode: str, jsonSchema: str = None, fi
         mg = ManifestGenerator(path_to_json_ld=self.inputMModelLocation, 
                                title=title, 
                                root=rootNode, 
-                               additional_metadata=additionalMetadata)
+                               additional_metadata=additionalMetadata,
+                               use_annotations=useAnnotations)
 
-        if jsonSchema:
-            return mg.get_manifest(json_schema=jsonSchema)
+        if datasetId and jsonSchema:
+            return mg.get_manifest(dataset_id=datasetId, json_schema=jsonSchema, sheet_url = sheetUrl)
+        
+        elif jsonSchema:
+            return mg.get_manifest(json_schema=jsonSchema, sheet_url = sheetUrl)
```

Comment:
I agree that this makes more sense. I took the format of the code before and just ran with it to avoid breaking stuff, but was wondering whether this was all necessary. Thanks for the review and revisions!